docs/testing.rst

```diff
@@ -1,21 +1,22 @@
 Testing
 =======
 
-django-timepiece includes several different alternatives for testing. Test can be run using the default django test runner, through `Tox <http://tox.testrun.org/latest/>`_, or with `django-jenkins <https://github.com/kmmbvnr/django-jenkins>`_. Tox and django-jenkins are not required to run the tests for django-timepiece, but it is possible to use them::
+Test requirements are listed `here <../requirements/tests.txt>`_.
 
-    pip install --upgrade tox django-jenkins
+You can run tests against your installation of django-timepiece using the
```

Comment:
Maybe clarify that these two methods only work in a full Django project where you've installed django-timepiece, and the last method works in the tree where you've checked out django-timepiece's source.